src/fiskaltrust.ifPOS/v2/Models/PayItem.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Runtime.Serialization;
+using fiskaltrust.Middleware.Localization.v2.Models.ifPOS.v2.Cases;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+#if NETSTANDARD2_1
+using System.Text.Json.Serialization;
+#endif
+
+namespace fiskaltrust.Middleware.ifPOS.v2.Models
+{
+    public class PayItem
+    {
+#if NETSTANDARD2_1
+        [JsonPropertyName("ftPayItemId")]
+#endif
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public Guid? ftPayItemId { get; set; }
+
+        private decimal _quantity = 1;
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("Quantity")]
+#endif
+        [DataMember(EmitDefaultValue = true, IsRequired = true)]
+        public decimal? Quantity
+        {
+            get => _quantity == 1 ? null : _quantity;
+            set => _quantity = value ?? 1;
+        }
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("Description")]
+#endif
+        [DataMember(EmitDefaultValue = true, IsRequired = true)]
+        public string Description { get; set; }
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("Amount")]
+#endif
+        [DataMember(EmitDefaultValue = true, IsRequired = true)]
+        public decimal Amount { get; set; }
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("ftPayItemCase")]
+#endif
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public PayItemCase ftPayItemCase { get; set; } = 0;
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("ftPayItemCaseData")]
+#endif
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public object? ftPayItemCaseData { get; set; }
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("Moment")]
+#endif
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public DateTime? Moment { get; set; }
+
+#if NETSTANDARD2_1
+        [JsonPropertyName("Position")]
+#endif
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public decimal Position { get; set; } = 0;
```

Comment:
@StefanKert Is the position a decimal so you can simply add something between two items without shifting all later items down?

I'd prefer that to be an integer if there's no special reason. it should be usable like an index I think...